blocks/collections-content-source-block/sources/content-api-collections.js

```diff
@@ -1,31 +1,64 @@
+import axios from 'axios';
+import { CONTENT_BASE, ARC_ACCESS_TOKEN } from 'fusion:environment';
 import getResizedImageData from '@wpmedia/resizer-image-block';
 
 const params = {
   _id: 'text',
   content_alias: 'text',
   from: 'text',
   size: 'text',
+  getNext: 'text',
 };
 
-const resolve = (key = {}) => {
+const fetch = (key = {}) => {
   const {
     'arc-site': site,
     _id,
     content_alias: contentAlias,
     from,
     size,
+    getNext = 'false',
   } = key;
   let updatedSize = size;
   // Max collection size is 20
   // See: https://redirector.arcpublishing.com/alc/docs/swagger/?url=./arc-products/content-api.json
   // Want to ensure size is capped at 20 to prevent an error.
   if (updatedSize && updatedSize > 9) updatedSize = size < 20 ? size : 20;
-  return `content/v4/collections?${_id ? `_id=${_id}` : `content_alias=${contentAlias}`}${site ? `&website=${site}` : ''}${from ? `&from=${from}` : ''}${size ? `&size=${updatedSize}` : ''}&published=true`;
+
+  return axios({
+    url: `${CONTENT_BASE}content/v4/collections?${_id ? `_id=${_id}` : `content_alias=${contentAlias}`}${site ? `&website=${site}` : ''}${from ? `&from=${from}` : ''}${size ? `&size=${updatedSize}` : ''}&published=true`,
```

Comment:
Do we need a `/` between `${CONTENT_BASE}` and `content/`?
I don't currently have a trailing `/` in the `CONTENT_BASE` in my `.env`. so I'm seeing the following:
![image](https://user-images.githubusercontent.com/452134/148404078-436ef75a-37b4-4971-8bf1-71a3031775b1.png)
I'm comparing this to `content-api-source-block` where we have a `/` preceding `content`.
https://github.com/WPMedia/arc-themes-blocks/blob/canary/blocks/content-api-source-block/sources/content-api.js#L11

Reply:
(I'll change my `.env` to include a trailing `/` in the `CONTENT_BASE` in the meanwhile to continue testing)

Reply:
That would depend on how the `CONTENT_BASE` is set up in a non local environment. I'll check with the Engine team

Reply:
Update the code to add the slash in, confirmed from Engine that the generated `CONTENT_BASE` will not contain it